e2e/babel-plugin-jest-hoist/__tests__/integration.test.js

```diff
@@ -15,7 +15,7 @@ import a from '../__test_modules__/a';
 import b from '../__test_modules__/b';
 import c from '../__test_modules__/c';
 import d from '../__test_modules__/d';
-import e from '../__test_modules__/e';
+import f from '../__test_modules__/f';
```

Comment:
`let f` below and keep `e` here?

Reply:
Doesn't make it better, as `f` is used in a different test etc. so that will make it look like a nice sequence here but mess it up down there :sweat_smile: